charmcraft.yaml

```diff
@@ -11,4 +11,9 @@ bases:
         channel: "20.04"
 parts:
   charm:
-    charm-binary-python-packages: [psycopg2-binary==2.9.3]
+    build-packages:
+      - libffi-dev
+      - libssl-dev
+      - rustc
+      - cargo
+    charm-binary-python-packages: [jsonschema, psycopg2-binary==2.9.3]
```

Comment:
thoughts on using `-` for a list (to stay consistent with the way we're listing build packages?

Reply:
Good suggestions, it is better to stay consistent with that. Updated on https://github.com/canonical/postgresql-operator/pull/28/commits/790e369813b6d0b86e8fe418750f091bc2b7b911.